prometheus-exporters/octobus-query-exporter/values.yaml

```
@@ -16,6 +16,8 @@ octobus_query_exporter:
         type: prometheus
       - name: scaleout
         type: thanos-ruler
+      - name: region
```

Please also adjust the folder from `region` to `regional`

```suggestion
      - name: regional
```